tests/test_snapshot_scheduler.cpp

```diff
@@ -233,7 +233,7 @@ BOOST_AUTO_TEST_CASE(snapshot_scheduler_old_json) {
    unsigned found = 0;
    for(const std::filesystem::directory_entry& dir_entry : std::filesystem::directory_iterator(temp / "snapshots"))
       found += std::regex_search(dir_entry.path().filename().string(), snapshotfile_regex);
-   BOOST_REQUIRE_GE(found, 3);
+   BOOST_REQUIRE_EQUAL(found, 3u);
```

Comment:
The test was originally more sloppy and it's why it was `>=3`; I forgot to change this to `==3` after making it more strict. Not a big enough deal to go back to 5.0 to fix this imo